docs/contributing.rst

```diff
@@ -64,8 +64,18 @@ Changing Test Requirements
 `.riot/requirements` contains requirements files generated with `pip-compile` for every environment specified by `riotfile.py`.
 Riot uses these files to build its environments, and they do not get rebuilt automatically when the riotfile changes.
 Thus, if you make changes to the riotfile, you need to run either `scripts/compile-and-prune-test-requirements` or `riot run -c <mytests>`
-to regenerate the requirements files for the environments that changed. You can commit and pull request changes to files in `.riot/requirements`
-alongside the corresponding changes to `riotfile.py`.
+to regenerate the requirements files for the environments that changed. In order to run the script, you need to have all minor versions of Python that the tracer supports.
+The easiest way to accomplish this and generate the files is to simply spin up the testagent container, exec into it, and run the script from there.
+
+.. code-block:: bash
+
+  cd dd-trace-py && docker run --network host --userns=host --rm -w /root/project -v $PWD/:/root/project \
+    -it ghcr.io/datadog/dd-trace-py/testrunner:1ed971833a2a3c97f43cbaeabcbb3f1e28745a00 \
```

Comment:
I think we can just use the testrunner.
```suggestion
  cd dd-trace-py && docker-compose run --rm testrunner \
```

Reply:
I picked that specific image because at the time it was faster than the default on my amd64 machine. I couldn't say without digging in why you're seeing that behavior.

Reply:
I'll add to backlog to look into